loadtest/control/actions.go

```diff
@@ -202,11 +202,25 @@ func CreatePost(u user.User) UserActionResponse {
 		return UserActionResponse{Err: NewUserError(err)}
 	}
 
-	postId, err := u.CreatePost(&model.Post{
+	post := &model.Post{
 		Message:   "Lorem ipsum dolor sit amet, consectetur adipiscing elit",
 		ChannelId: channel.Id,
 		CreateAt:  time.Now().Unix() * 1000,
-	})
+	}
+
+	// 85% of posts are replies
+	if rand.Float64() < 0.85 {
```

Comment:
This number is taken from https://community.mattermost.com/private-core/pl/4otj7nxwxfb39q9ywp334gqzso, but I'm waiting for the results of the more general query requested in https://community.mattermost.com/private-core/pl/c89zba5dr3dutfjwks793ot9go

Reply:
This change is one of the main reasons I originally implemented it the way it is. If we start adding all these probability checks inside the generic actions we are essentially turning everything into a simulcontroller. In hindsight that may be okay as that's probably the only truly useful controller but mostly sharing a bit of context.

Reply:
Perhaps we can pass an additional param as `controllerType` and do this _only_ when it is a simulcontroller? Or otherwise, we can have a dedicated simulcontroller specific createPost and a generic createPost for simpleController.

But I agree that adding probability checks inside the top-level package functions is not a good idea.